monkestation/code/modules/donator/code/datum/loadout.dm

```diff
@@ -248,9 +244,45 @@
 	requires_purchase = FALSE
 
 //zedmastermind
-/datum/loadout_item/head/zed_officercap
+/datum/loadout_item/head/zed_offic/datum/loadout_item/suit/aotcloak
+	name = "Survey Corps Cloak"
+	item_path = /obj/item/clothing/suit/hooded/aotcloak
+	donator_only = TRUE
+	requires_purchase = FALSE
 	name = "Zed Officer Cap"
 	item_path = /obj/item/clothing/head/costume/zed_officercap
 	donator_only = TRUE
 	requires_purchase = FALSE
-	ckeywhitelist = list("zedmastermind")
```

Comment:
I... don't think you meant to move the aotcloak to another part of the file - and I especially don't think you meant to remove the ckeywhitelist for the zed_officercap.